components/admin_console/system_user_detail/index.ts

```diff
@@ -23,10 +27,10 @@ function mapStateToProps(state, ownProps) {
         mfaEnabled: config.EnableMultifactorAuthentication === 'true',
         navigationBlocked: getNavigationBlocked(state),
         showNavigationPrompt: showNavigationPrompt(state),
-    };
+    } as any;
```

Comment:
Why this is any? souldn't it be Props or at least `Partial<Props>`

Reply:
Type checking is complaining about actions being incompatible and I was not able to figure out how to fix that.
Could you take a look?